vnext/Microsoft.ReactNative/GlyphViewManager.h

```diff
@@ -18,7 +18,7 @@ class GlyphViewManager : public FrameworkElementViewManager {
   void GetNativeProps(const winrt::Microsoft::ReactNative::IJSValueWriter &writer) const override;
 
  protected:
-  XamlView CreateViewCore(int64_t tag) override;
+  XamlView CreateViewCore(int64_t tag, winrt::Microsoft::ReactNative::JSValueObject &props) override;
```

Comment:
> &props [](start = 83, length = 7)

I wonder why do we pass props by reference. It should be either const& if we do not plan to take the ownership, or && if we do.

Reply:
good catch, fixed